mmv1/products/dataplex/Glossary.yaml

```diff
@@ -0,0 +1,121 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Glossary'
+min_version: beta
+description: |
+  Represents a collection of categories and terms defined by the user. Glossary is a top level resource and is the GCP parent resource of all the categories and terms within it.
+docs:
+base_url: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/glossaries?glossary_id={{glossary_id}}'
+update_verb: 'PATCH'
+update_mask: true
+timeouts:
+  insert_minutes: 5
+  update_minutes: 5
+  delete_minutes: 5
+autogen_async: true
+import_format: ['projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}']
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+iam_policy:
+  exclude_import_test: true
+  method_name_separator: ':'
+  parent_resource_attribute: 'glossary_id'
+  example_config_body: 'templates/terraform/iam/iam_attributes.go.tmpl'
+  import_format:
+    - 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+custom_code:
+examples:
+  - name: 'dataplex_glossary_basic'
+    min_version: beta
+    primary_resource_id: 'glossary_test_id'
+    primary_resource_name: 'fmt.Sprintf("tf-test-glossary%s", context["random_suffix"])'
+    vars:
+      glossary_name: 'glossary-basic'
+    test_env_vars:
+      project_name: 'PROJECT_NAME'
+  - name: 'dataplex_glossary_full'
+    min_version: beta
+    primary_resource_id: 'glossary_test_id_full'
+    primary_resource_name: 'fmt.Sprintf("tf-test-glossary%s", context["random_suffix"])'
+    vars:
+      glossary_name: 'glossary-full'
+    test_env_vars:
+      project_name: 'PROJECT_NAME'
+parameters:
+  - name: 'location'
+    type: String
+    description: |
+      The location where the glossary should reside.
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'glossaryId'
+    type: String
+    description: |
+      The glossary id for creation.
+    url_param_only: true
+    immutable: true
+properties:
+  - name: 'name'
+    type: String
+    description: |
+      The resource name of the Glossary. Format: projects/{projectId}/locations/{locationId}/glossaries/{glossaryId}
+    output: true
+  - name: 'displayName'
+    type: String
+    description: |
+      User friendly display name of the glossary. This is user-mutable. This will be same as the glossaryId, if not specified.
+    required: false
```

Comment:
nit: here & below: `required: false` is the default so you don't need to state it. (But I wouldn't block on this.)